### PR TITLE
MAP-1858 helm_deploy aligned with Template Typescript

### PR DESCRIPTION
**Pull request by marcusaleman**
[WPIP UI] Use git cherry pick from template project to do security update in Frontend

It's added only helm_deploy changes from Template Typescript in this Pull request.

		Note promoetehus alerts aligned with templates and other projects.
		Note Appinsight aligned with latest templates changes.

### CODE REVIEW

File: helm_deploy/values-preprod.yaml [hunk 1]
```diff
@@ -34,3 +34,6 @@ generic-service:
     SHOW_PRISON_TRANSFER_SUMMARY: 'false'
     SHOW_BREADCRUMB: 'true'
     SHOW_RECENT_ARRIVALS: 'true'
+
+generic-prometheus-alerts:
```
Comment (marcusaleman) [inline helm_deploy/values-preprod.yaml:38]: This label align with pre-prod env promoethus alerts and are introduced new in the WPIP-UI

File: helm_deploy/hmpps-welcome-people-into-prison-ui/Chart.yaml [hunk 2]
```diff
@@ -5,8 +5,8 @@ name: hmpps-welcome-people-into-prison-ui
 version: 0.2.0
 dependencies:
   - name: generic-service
-    version: "2.8"
+    version: "3.7"
     repository: https://ministryofjustice.github.io/hmpps-helm-charts
   - name: generic-prometheus-alerts
-    version: 1.1.0
+    version: "1.11"
```
Comment (marcusaleman) [inline helm_deploy/hmpps-welcome-people-into-prison-ui/Chart.yaml:11]: Needed Chart values update

File: helm_deploy/hmpps-welcome-people-into-prison-ui/values.yaml [hunk 3]
```diff
@@ -55,6 +54,8 @@ generic-service:
     elasticache-redis:
       REDIS_HOST: "primary_endpoint_address"
       REDIS_AUTH_TOKEN: "auth_token"
+    application-insights:
```
Comment (marcusaleman) [inline helm_deploy/hmpps-welcome-people-into-prison-ui/values.yaml:57]: there is a new template structure to add appinsights variables

**Review by GurnankCheema — APPROVED** [review]
LGTM

**Pull request merged**